content/open-source/wolfi/overview.md

```diff
@@ -35,3 +35,41 @@ Wolfi, whose name was inspired by the [world's smallest octopus](https://en.wiki
 - Designed to support glibc and musl
 
 Wolfi enables Chainguard to solve the software supply chain security problem from the outside in. It gives developers the secure-by-default base they need to build software, it scales to support organizations running massive environments and provides the control needed to fix most modern supply chain threats. Wolfi builds all packages directly from source, allowing us to fix vulnerabilities or apply customizations that improve the supply chain security posture of everything from compilers to language package managers.
+
```

Comment:
This is in #958 and was reviewed there, can you remove this from this PR?

Reply:
Sorry, my bad. I actually made a mess of #958 because it's on the main branch of my fork.

Reply:
Should be fixed now.